package.json

```diff
@@ -34,6 +34,7 @@
     "commander": "^2.15.1",
     "lodash": "^4.17.5",
     "npm-programmatic": "0.0.10",
+    "semver": "^5.5.0",
```

Comment:
There were no updates on the `package-lock`? It seems we were already depending on this one